frontend/tests/dag-table-test.ts

```diff
@@ -2612,11 +2613,10 @@ test.describe("dag table tests", () => {
 
   test.describe("horizontal scrollbar", () => {
     test("detect the presence of a horizontal scrollbar", async ({ page }) => {
-      const hasHorizontalScrollbar = await page.evaluate(() => {
-        return document.body.scrollWidth > document.body.clientWidth;
-      });
-
-      expect(hasHorizontalScrollbar).toBe(false);
+      await expect(hasHorizontalScrollbar(page)).toBe(false);
```

Comment:
Check the error message in playwright.

You need to await the result of your function:

```suggestion
      expect(await hasHorizontalScrollbar(page)).toBe(false);
```

Reply:
it worked!